resources/views/pages-legacy/gameInfo.blade.php

```diff
@@ -616,8 +631,15 @@ function resize() {
                 echo "<div class='text-right'><button class='btn'>Submit</button></div>";
                 echo "<label for='game_genre'>Genre</label><input type='text' name='genre' id='game_genre' value='" . attributeEscape($genre) . "' class='w-full'>";
                 echo "<div class='text-right'><button class='btn'>Submit</button></div>";
-                echo "<label for='game_release'>First Released</label><input type='text' name='release' id='game_release' value='" . attributeEscape($released) . "' class='w-full'>";
-                echo "<div class='text-right'><button class='btn'>Submit</button></div>";
+                ?>
+                
+                <label for='game_release'>First Released</label>
+                <input type="date" name="release" id="game_release" value="{{ $gameModel->released_at->format('Y-m-d') }}" class="w-full">
+                <div class="text-right">
+                    <button class="btn">Submit</button>
```

Comment:
Changing the date via this widget forces the released_at_granularity to day.

It also creates an audit comment instead of an audit log entry.

Maybe these fields should be removed from here as they become available in filament?

Reply:
I'm on board with removing them. Done in latest.